marklogic-data-hub/src/test/java/com/marklogic/hub/flow/TemporalWriteTest.java

```diff
@@ -0,0 +1,128 @@
+package com.marklogic.hub.flow;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.node.ArrayNode;
+import com.fasterxml.jackson.databind.node.ObjectNode;
+import com.marklogic.appdeployer.command.temporal.DeployTemporalAxesCommand;
+import com.marklogic.appdeployer.command.temporal.DeployTemporalCollectionsCommand;
+import com.marklogic.appdeployer.impl.SimpleAppDeployer;
+import com.marklogic.client.io.DocumentMetadataHandle;
+import com.marklogic.client.io.JacksonHandle;
+import com.marklogic.hub.AbstractHubCoreTest;
+import com.marklogic.hub.HubConfig;
+import com.marklogic.mgmt.resource.databases.DatabaseManager;
+import com.marklogic.mgmt.util.ObjectMapperFactory;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import static com.marklogic.client.io.DocumentMetadataHandle.Capability.*;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class TemporalWriteTest extends AbstractHubCoreTest {
+
+    @BeforeEach
+    void setUp() {
+        runAsDataHubDeveloper();
+        installProjectInFolder("temporal-test");
```

Comment:
Can you move the new project files to test-projects/temporal-test? I'd eventually like for all of the test projects to be in that folder so that we don't have so many folders directly under src/test/resources.